A Beginner's Guide to Umbraco Commerce

eshanrnh opened the pull request:
## Description

A guide to setting up Umbraco Commerce, including an introduction to basic concepts and configuration.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [x] New content
* [x] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

v14

## Deadline (if relevant)

Anytime

_When should the content be published?_

sofietoft commented on 14/umbraco-commerce/tutorials/getting-started-with-commerce.md:
```diff
@@ -0,0 +1,183 @@
+
+## Introduction
+
+Umbraco Commerce (formerly known as Vendr) is an eCommerce platform that integrates with Umbraco CMS. It offers features such as product management, order processing, and payment integrations. This guide will walk you through setting up Umbraco Commerce.
```
I don't think we should mention Vendr anymore.

```suggestion
Umbraco Commerce is an eCommerce platform that integrates with Umbraco CMS. It offers features such as product management, order processing, and payment integrations. This guide will walk you through setting up Umbraco Commerce.
```

sofietoft commented on 14/umbraco-commerce/tutorials/getting-started-with-commerce.md:
```diff
@@ -0,0 +1,183 @@
+
+1. Navigate to the **Settings** section.
+2. Click **+** next to **Stores**.
+3. Enter a **Name** for the Store. For this tutorial, I'll call it *Products*.
```
I'd suggest calling the Store something different.
I think having a store called Products might be confusing when having to describe working with actual products? 😅 

Suggestions for store names could be: "Umbraco Swag Store" or "The Umbraco Store"

eshanrnh replied:
I have changed it to _Umbraco Swag Store_ and will update all the images to reflect the same 💪

sofietoft commented on 14/umbraco-commerce/tutorials/getting-started-with-commerce.md:
```diff
@@ -0,0 +1,183 @@
+
+    ![Create Store](images/create-store.png)
+4. Click **Create**.
+5. Configure the **General**, **Notification**, **Order**, **Product**, and **Gift Card Settings** in the **Details** tab.
+6. Select the **User Roles** and **Users** that must have access to manage the Store in the **Permissions** tab.
```
A suggestion to make these steps even clearer:

```suggestion
5. Open the **Details** workspace view.
6. Configure the **General**, **Notification**, **Order**, **Product**, and **Gift Card Settings**.
7. Open the **Permissions** workspace view.
8. Select the **User Roles** and **Users** that should have access to manage the Store.
```

sofietoft replied:
Actually, I'm reconsidering this part.
It doesn't make sense to configure any of these, before actually setting something up following the rest of the article first, right? 🤔

sofietoft replied:
OR, it could make sense to mention this part again at the end, when all the "Setting up" sections have been run through? 🤔

eshanrnh replied:
Yes, even I thought to mention this part at the end once all the other setup is done as then those appear in the drop-down for this part :D Will move this part to the end.

sofietoft commented on 14/umbraco-commerce/tutorials/getting-started-with-commerce.md:
```diff
@@ -0,0 +1,183 @@
+
+### Setting up a Location
+
+1. Navigate to the **Settings** section.
```
Aren't we already in the **Settings** section based on the section above?

eshanrnh replied:
Good point 😀

sofietoft commented on 14/umbraco-commerce/tutorials/getting-started-with-commerce.md:
```diff
@@ -0,0 +1,183 @@
+
+### Setting up Order Status
+
+When you first set up Umbraco Commerce, it comes with a set of predefined order statuses to help manage the order lifecycle. These statuses usually include: New, Completed, Cancelled, and Error. These statuses can be customized depending on your specific business requirements.
```
```suggestion
When you first set up Umbraco Commerce, it comes with predefined order statuses to help manage the order lifecycle. These statuses include New, Completed, Cancelled, and Error. These statuses can be customized depending on your specific business requirements.
```

sofietoft commented on 14/umbraco-commerce/tutorials/getting-started-with-commerce.md:
```diff
@@ -0,0 +1,183 @@
+1. Navigate to the **Settings** section.
+2. Go to **Payment Methods** under the Store.
+3. Click **Create Payment Method**.
+4. Select a payment provider from the list. For Example: *Zero Value*.
```
```suggestion
4. Select a payment provider from the list. For example: *Zero Value*.
```

sofietoft commented on 14/umbraco-commerce/tutorials/getting-started-with-commerce.md:
```diff
@@ -0,0 +1,183 @@
+    ![Create Shipping Method](images/create-shipping-method.png)
+7. Click **Save**.
+
+### Setting up a Country
```
What does it mean to set up a country?
Is this to indicate which country products are sold to?
Would it make sense to add some more info to this part?

Also, how is country different from location?

sofietoft commented on 14/umbraco-commerce/tutorials/getting-started-with-commerce.md:
```diff
@@ -0,0 +1,183 @@
+
+![Built-in Property Editors](images/property-editors.png)
+
+Some of the property editors include:
```
```suggestion
The available property editors include:
```

sofietoft reviewed (commented):
This is a really great guide that covers all the basics of setting up Umbraco Commerce in the backoffice 💪 
I've added some suggestions - mostly minor things.

Also, I have some general thoughts:

- Should we add a disclaimer about how this guide DOES NOT cover building all the pages and the functionality behind the configuration set in the backoffice?
- Are taxes required to set up? They might be, I just don't know.
- I'm missing some more text in the last "Setting up" sections. They just start with a list, but I think it would be nice with some text explaining what will be setup in the given section. Perhaps a link out to other articles covering the topic?

sofietoft reviewed (commented):
Should each of the "Setting up" sections start with the following instead:

1. Select your store from the **Store** menu in the **Settings** section
2. Open the **xxxx** menu.

I think perhaps "the Store" could become a bit too generic?

eshanrnh commented:
Thanks @sofietoft for your review. 🙌 I believe I have incorporated all your comments. I have refined the guide a bit more to flow smoothly and added all the missing bits and pieces as pointed out. I also added the Templates part which was not covered earlier.

Lastly, I am wondering if the Title is fine or should we go with **Getting Started with Umbraco Commerce**

sofietoft commented on 14/umbraco-commerce/tutorials/getting-started-with-commerce.md:
```diff
@@ -0,0 +1,229 @@
+This tutorial focuses exclusively on setting up Umbraco Commerce, introducing key concepts, and the configuration process. It has been tested on the latest releases of **Umbraco CMS version 14** and **Umbraco Commerce version 14**.
+
+{% hint style="info" %}
+This tutorial does not provide instructions for building a complete eCommerce website.
```
I know something similar is mentioned above, but just to make it even clearer - how about adding something like this?

```suggestion
This tutorial does not provide instructions for building a complete eCommerce website. It focusses solely on the setup through the Umbraco backoffice.
```

sofietoft reviewed (commented):
This is looking really great!
Great idea giving it a mores smooth flow - it all feels a bit more connected now 👏 

I've added a single suggestion to the disclaimer in the top.
Also, one more thing:
I think the images should be on their own line, and not part of the list. 
When they're part of the list, they're placed a little to the right, which makes it a look a little strange.
I know this is a matter of consistency, but for that, I vote for only have the text in the lists 🙈 

As for the title of the article, how about making it a bit more concrete?
Something like "Configure Umbraco Commerce in the backoffice" or "Getting started with Umbraco Commerce: The Backoffice"?
Just so people don't expect to find a full tutorial for building a complete shop 😄

eshanrnh commented:
i've incorporated the title and image indentation suggestion. Let me know what you think, @sofietoft 💪

sofietoft reviewed (approved):
Looks great!! 👏 😄
